static/app/components/forms/fields/textareaField.tsx

```diff
@@ -30,20 +28,31 @@ function TextareaField({
         children: React.ReactNode;
         model: FormModel;
         name: string;
-      }) => (
-        <InputGroup>
-          <InputGroup.TextArea
-            {...{monospace, rows, autosize, name}}
-            // Do not forward required to `textarea` to avoid default browser behavior
-            {...omit(fieldProps, ['onKeyDown', 'children', 'required'])}
-          />
-          {!hideControlState && (
-            <InputGroup.TrailingItems>
-              <FormFieldControlState model={model} name={name} />
-            </InputGroup.TrailingItems>
-          )}
-        </InputGroup>
-      )}
+      }) => {
+        const {
+          onKeyDown: _,
+          children: _c,
+          required: _r,
+          ...rest
+        } = fieldProps as Record<any, any>;
```

Comment:
Can we avoid having any in here?

Reply:
I can type as object, but the issue comes from bad types which end up being typed as {} so we cant destructure due to missing index signature